cvat/apps/engine/views.py

```diff
@@ -887,23 +959,36 @@ def upload_finished(self, request):
             task_data = self._object.data
             serializer = DataSerializer(task_data, data=request.data)
             serializer.is_valid(raise_exception=True)
-            data = dict(serializer.validated_data.items())
-            uploaded_files = task_data.get_uploaded_files()
```

Comment:
This appears to be the only usage of `get_uploaded_files`, so it can probably be deleted.

Reply:
Yes, done

Reply:
Yes, done